salt/modules/rbenv.py

```diff
@@ -415,13 +419,16 @@ def do_with_ruby(ruby, cmdline, runas=None):
         raise SaltInvocationError('Command must be specified')
 
     try:
-        cmdline = shlex.split(cmdline)
+        if isinstance(cmdline, basestring):
```

Comment:
If it will depend on #28704 then I will update it to use the util version I
made.
On Nov 9, 2015 1:03 PM, "Colton Myers" <notifications@github.com> wrote:

> In salt/modules/rbenv.py
> <https://github.com/saltstack/salt/pull/28702#discussion_r44315316>:
>
> > @@ -415,13 +419,16 @@ def do_with_ruby(ruby, cmdline, runas=None):
> >          raise SaltInvocationError('Command must be specified')
> >
> >      try:
> > -        cmdline = shlex.split(cmdline)
> > +        if isinstance(cmdline, basestring):
>
> salt.ext.six.string_types instead of basestring
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/saltstack/salt/pull/28702/files#r44315316>.
>